build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.1.2'
```

Comment:
_noice.jpeg_

can you tell us or maybe link some info what it gives us? faster builds?

Reply:
https://tomgregory.com/top-gradle-7-features-and-improvements/
and also to make properly work latest Api 31 and latest kotlin version and java 11 we need this gradle update =)